internal/ykclient/client.go

```diff
@@ -36,9 +33,18 @@ var (
 	// nodeEndPt  = func(partitionName, nodeID string) string {
 	// 	return fmt.Sprintf("/ws/v1/partition/%s/node/%s", partitionName, nodeID)
 	// }
-	nodeUtilEndPt     = "/ws/v1/scheduler/node-utilizations"
-	applicationsEndPt = func(partitionName, queueName string) string {
-		return fmt.Sprintf("/ws/v1/partition/%s/queue/%s.%s/applications", partitionName, queueName, partitionName)
```

Comment:
I checked with creating multiple queues, it seems that the partition name isn't included in the queue path. Usually when we test our setup with a single queue, `root`, Yunikorn creates a default leaf queue called `root.default` because jobs can only be submitted to a leaf queue so it creates `root.default` as a default behaviour.